src/frontend/js/components/DashboardAddressesManagement/AWindowMock.tsx

```diff
@@ -0,0 +1,15 @@
+window.__richie_frontend_context__ = {
+  context: {
+    authentication: {
+      backend: 'dummy',
+      endpoint: 'http://localhost:8073',
+    },
+    csrftoken: '',
+    environment: 'development',
+    joanie_backend: {
+      endpoint: 'http://localhost:8071/',
+    },
+    release: 'NA',
+    sentry_dsn: null,
+  },
+};
```

Comment:
Not a fan to use file to create mock like that. Even if this is more verbose, I prefer to declare this mock next to the test suite.